content/docs/command-reference/get.md

```diff
@@ -37,10 +37,11 @@ to an "offline" repo (if it's a DVC repo without a default remote, instead of
 downloading, DVC will try to copy the target data from its <abbr>cache</abbr>).
 
 The `path` argument is used to specify the location of the target to be
-downloaded within the source repository at `url`. `path` can specify any file or
-directory in the source repo, including those tracked by DVC, or by Git. Note
-that DVC-tracked targets should be found in a `dvc.yaml` or `.dvc` file of the
-project.
+downloaded within the source repository at `url`, including those nested inside
+directories [tracked as a whole](/doc/command-reference/add#example-directory).
```

Comment:
inside [tracked directories ](link) ... again, not sure if `add` is the correct link

Reply:
`add` is where we explain the mechanics of DVC file/dir tracking.